### DISCUSSION TITLE
Image Classification on Caltech101 dataset

### DISCUSSION THREAD

**Discussion by RahulKannaAV** [mrdbourke/pytorch-deep-learning#1118] [Show and tell]
Hello everyone. I hope you are doing well. I'm excited to showcase my first deployed model (excluding the coursework models), based on Image Classification problem. I've used ViT Base Feature Extractor model and Caltech101 dataset to classify Objects.

I am eager to hear your suggestions as well as project ideas to explore further into the field of Deep Learning

Link                                                :  https://huggingface.co/spaces/itzRahul/ObjectVision_with_ViT
Percentage of Data Used               :   100%
Accuracy obtained after 5 Epochs :  94%

**Comment by SRIKANTHDOSU** [comment]
Hello brother,
Which topics are used to create this kind of project

**Reply by RahulKannaAV** [reply]
I've referred to Computer Vision section for creating dataloaders

Transfer Learning section to create feature extractor model

Model Deployment section to deploy the model in HuggingFace Spaces with Gradio UI.

This follows general procedure to solve Image Classification problem
